workspace/resource_notebook.go

```diff
@@ -131,10 +131,10 @@ func (a NotebooksAPI) Mkdirs(path string) error {
 // List will list all objects in a path on the workspace
 // and with the recursive flag it will recursively list
 // all the objects
-func (a NotebooksAPI) List(path string, recursive bool) ([]ObjectStatus, error) {
+func (a NotebooksAPI) List(path string, recursive bool, ignoreErrors bool) ([]ObjectStatus, error) {
```

Comment:
please make a PR to SDK

Reply:
Will do separately - right now it's blocking some PS work... Really I need to think a bit about that List APIs